File: SoftUni - C# OOP/Interfaces and Abstraction/Interfaces and Abstraction - Exercise/04. Border Control/Citizen.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace _04._Border_Control
{
    public class Citizen : ICitizens
    {
        public string Name {  get; set; }

        public int Age { get; set; }

        public string Id { get; set; }

        public Citizen(string name,int age, string id)
        {
            this.Name = name;
            this.Age = age;
            this.Id = id;
        }
    }
}

File: SoftUni - Java OOP/Interfaces and Abstraction/Interfaces and Abstraction - Lab/_05_Border_Control/Citizen.java
package _05_Border_Control;

public class Citizen implements Identifiable {
    private String name;
    private int age;
    private String id;

    public Citizen(String name, int age, String id) {
        this.setName(name);
        this.setAge(age);
        this.setId(id);
    }
    @Override
    public String getId() {
        return this.id;
    }

    public String getName() {
        return name;
    }
    public int getAge() {
        return age;
    }

    public void setName(String name) {
        this.name = name;
    }

    public void setAge(int age) {
        this.age = age;
    }
    public void setId(String id) {
        this.id = id;
    }
}
